include/z64.h

```diff
@@ -700,6 +700,22 @@ typedef struct {
     /* 0x344 */ u32 unk_344;
 } InterfaceContext; // size = 0x348
 
+typedef struct {
+    /* 0x00 */ void* loadedRamAddr;
+    /* 0x04 */ u32 vromStart;
+    /* 0x08 */ u32 vromEnd;
```

Comment:
```suggestion
    /* 0x04 */ void* vromStart;
    /* 0x08 */ void* vromEnd;
```

Reply:
idk i personally feel like these should be uintptr_t because I don't think its pointing to an actual loaded area in memory, just rom